internal/notifier/notifier.go

```diff
@@ -135,7 +137,7 @@ func (n *Notifier) getConnAndRun(ctx context.Context) {
 		if errors.Is(err, context.Canceled) {
 			return
 		}
-		log.Printf("error establishing connection from pool: %v", err)
+		slog.Error("error establishing connection from pool", "err", err)
```

Comment:
I think this invocation and subsequent ones should be using the logger instance set above right?

Reply:
yes 🤦🏼‍♂️